src/libraries/System.Private.CoreLib/src/System/Threading/Interlocked.cs

```diff
@@ -192,38 +192,36 @@ public static float Exchange(ref float location1, float value)
         public static double Exchange(ref double location1, double value)
             => Unsafe.BitCast<long, double>(Exchange(ref Unsafe.As<double, long>(ref location1), Unsafe.BitCast<double, long>(value)));
 
-        /// <summary>Sets a platform-specific handle or pointer to a specified value and returns the original value, as an atomic operation.</summary>
+        /// <summary>Sets a native-sized signed integer to a specified value and returns the original value, as an atomic operation.</summary>
```

Comment:
This repo is not the source of truth for the documentation for CoreLib. Could you please submit the same edit to http://github.com/dotnet/dotnet-api-docs/?

Reply:
https://github.com/dotnet/dotnet-api-docs/pull/11643